### ISSUE TITLE
Meaning and values of passive components before balun

### ISSUE THREAD

**Issue by kolins-cz** [cernohorsky/AntennaBalun#2]
Hello,
I am glad you found my design and re-used it. But could you please explain meaning of those passive components?
Are they for a filter or for better impedance match? Either way, could you please state frequency you are using and component values?

Thanks
Mirek

**Comment by cernohorsky** [comment]
Exactly, the component are there for a impedance matching. Or it can be fitted with a simple lowpass or highpass filter or used as a DC-block to improve the S/N ratio. In fact, I've never used any of it. The difference in impedance between a dipole and a 50 ohm cable is minimal. Moreover, the solution of matching is always frequency dependent and I never wanted always to search for correct components. The final result is not worth it in my opinion.
But thank you that I can reuse your design. You did a great job!
